libr/util/assert.c

```diff
@@ -0,0 +1,12 @@
+#include <r_util.h>
+
+/*
+ * It prints a message to the log and it provides a single point of entrance in
+ * case of debugging. All r_return_* functions call this.
+ */
+R_API void r_assert_log(RLogLevel level, const char *fmt, ...) {
```

Comment:
i dont get the point on having a 12LOC file with just one useless function that is a copy of vlogf and at the same time refuse to have the strings for warnings in a single place, which will be the right place in here

Reply:
This function is used by the "assert" functions, so that in case you see one of them, you can use gdb and break on this function. If we don't have this, but we just directly call `r_log`, then if `r_log` is used in other places you may have an hard time breaking exactly at the broken place.

Reply:
The reasons to not move the strings inside another function is because, for example, for https://github.com/radare/radare2/pull/11286/files#diff-7ae8990ae6b037702a50871807f77330R81 and for https://github.com/radare/radare2/pull/11286/files#diff-7ae8990ae6b037702a50871807f77330R96 I'd need two separate functions since they have different strings.